frontend/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,18 @@
+import { useRef } from 'react'
+import { Tab as ChakraTab, TabProps } from '@chakra-ui/react'
+
+export const Tab = (props: TabProps): JSX.Element => {
+  const tabRef = useRef<HTMLButtonElement>(null)
+
+  // Ensures that when navigating horizontally through tabs,
+  // the focused tab is scrolled horizontally into view
+  const handleFocus = () => {
+    tabRef.current?.scrollIntoView({
+      // center horizontally
+      inline: 'center',
+      // prevent unnecessary vertical scrolling
+      block: 'nearest',
+    })
```

Comment:
This block is causing the display to "jump" when you click on a tab that is already in the viewport; maybe it's due to `use-drag-scroll`?

Reply:
isn't this the behaviour you requested in [this comment](https://github.com/opengovsg/FormSG/pull/2304#pullrequestreview-705795355)?

Reply:
yea, but it's jumping even when the thing is in the viewport, but does not happen all the time, weirdly enough. maybe remove the `inline` prop so it doesn't jump to the center unnecessarily once it's in the viewport?

Reply:
sure, done in 3c8229d